taxonomies/labels.txt

```diff
@@ -560,12 +619,14 @@ nl:Max Havelaar
 wikidata:en:Q694008
 
 <en:Max Havelaar
+de:Max Havelaar Belgien
```

Comment:
The first entry is the canonical entry, it defines what will be the canonical id. So it should not be de:something, unless it is a label that exists only in Germany.